### PR TITLE
[JA] Shipping privacy sandbox

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/59481)

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
CC: @agektmr

File: site/ja/blog/shipping-privacy-sandbox/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+---
+layout: layouts/blog-post.njk
+title: プライバシーサンドボックスの関連性と測定 API の出荷の準備
+authors:
+  - rowan_m
+  - alexandrawhite
+description: これらの API の Chrome での有効化は、7 月下旬の Chrome 安定版 115 で開始されます。
```
Comment (agektmr) [inline site/ja/blog/shipping-privacy-sandbox/index.md:7]:
```suggestion
description: これらの API の Chrome での提供は、7 月下旬の Chrome 安定版 115 で開始されます。
```

File: site/ja/blog/shipping-privacy-sandbox/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+この記事では、今回のリリースに関わる次の内容について確認します。
+
+- **出荷内容**。公開予定の関連性と測定 API は、Topics、Protected Audience、アトリビューション レポート、Private Aggregation、共有ストレージ、および Fenced Frames です。これらの API は、潜在的なイシューを監視するために段階的に利用可能になる予定です。
+- **正式な公開プロセス**。各 API は標準の Chrome 公開プロセスをたどります。このプロセスには、blink-dev メーリングリストに承認を得る目的で公開される個別の「Intent to Ship」メッセージが含まれます。
+- **ユーザーコントロールの更新**。ユーザーは、API を管理するための広告プライバシーコントロールを利用できます。
+- **オリジントライアルのステータス**。オリジントライアルは安定版リリースまで引き続き利用可能です。
+- **登録**。登録は 6 月に開始され、8 月には関連性 API と測定 API にアクセスできる必要があります。
+- **Chrome を使用したテスト**。開発者がサードパーティ Cookie データを使用せずに API をテストできるオプションを準備しています。
+
+Chrome での有効化が近づきましたら、改めてお知らせいたします。今のところ、開発者がすぐにできる唯一のアクションは、情報を入手することです。これから起こる変更を理解しておくことで、サイトの準備を確実に行うことができます。
```
Comment (agektmr) [inline site/ja/blog/shipping-privacy-sandbox/index.md:28]:
```suggestion
Chrome での提供が近づきましたら、改めてお知らせいたします。今のところ、開発者がすぐにできる唯一のアクションは、情報を入手することです。これから起こる変更を理解しておくことで、サイトの準備を確実に行うことができます。
```

File: site/ja/blog/shipping-privacy-sandbox/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+「Chrome での有効化」とは、ブラウザのフラグやオリジントライアルへの参加を必要とせずに、デフォルトの Chrome で API をで利用できることを意味します。ただし、100% すべての Chrome ブラウザで API がすぐに有効になるということではありません。API は段階的に公開され、API がアクティブであるかどうかはユーザーがいつでも制御できます。立ち上げが完了すると、エコシステムは本番環境で API を使用できるようになります。
```
Comment (agektmr) [inline site/ja/blog/shipping-privacy-sandbox/index.md:30]:
```suggestion
「Chrome での提供」とは、ブラウザのフラグやオリジントライアルへの参加を必要とせずに、デフォルトの Chrome で API をで利用できることを意味します。ただし、100% すべての Chrome ブラウザで API がすぐに有効になるということではありません。API は段階的に公開され、API がアクティブであるかどうかはユーザーがいつでも制御できます。立ち上げが完了すると、エコシステムは本番環境で API を使用できるようになります。
```

**Review by agektmr — CHANGES_REQUESTED** [review]
We've changed the wording for this.

**Review by agektmr — CHANGES_REQUESTED** [review]
We've changed the wording for this.

**Comment by ilyaspiridonov** [comment]
@agektmr done

**Review by agektmr — APPROVED** [review]

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**